CIP-066/README.md

```diff
@@ -0,0 +1,426 @@
+The only important thing to note is that inside the DID document the payload will contain the different social accounts that were linked to the DID document.
+
+```
+{
+  "@context": [
+    "https://www.w3.org/ns/did/v1",
+    "https://w3id.org/security/suites/ed25519-2020/v1"
+  ],
+  "payload": {
+    ...
+  },
+  "id": "did:example:123456789abcdefghi",
+  "authentication": [{
+    
+    "id": "did:example:123456789abcdefghi#keys-1",
+    "type": "Ed25519VerificationKey2020",
+    "controller": "did:example:123456789abcdefghi",
+    "publicKeyMultibase": "zH3C2AVvLMv6gmMNam3uVAjZpfkcJCwDwnZn6z3wXmqPV"
+  }]
+}
+```
+
+#### 2. Collection Token
+
+This is an unnamed token, as first described in the [CIP-0027](hhttps://github.com/cardano-foundation/CIPs/tree/master/CIP-0027).
+
+It stores information about the location of either the DID or the Identity Token in the metadata of its minting transaction.
+
+```
+{
+  "725": {
+    "<policy_id>": {
+        "<collection_name>": {
+            "@context": <uri | array>,
+            "type": <string>,
+            "files": [{
+                "name": <string>,
+                "mediaType": <mime_type>,
+                "src": <string | array>,
+                <other_properties>
+            }],
+            <other_properties>
+        }
+    },
+    "version": <version_id>
+  }
+}
+```
+
+| identifier                  | description                                                         | optional |
+| --------------------------- | ------------------------------------------------------------------- | -------  |
+| policy_id                   | The policy of the NFT collection                                    | no       |
+| collection_name             | The name of the NFT collection                                      | no       |
+| @context                    | Rerefences to the DID standard                                      | yes      |
+| type                        | Verification method type from W3C specification                     | no       |
+| name                        | Name of the DID                                                     | yes      |
+| mediaType                   | The mediaType of the referenced file                                | no       |
+| src                         | A link to the DID file or an asset_fingerprint to an Identity Token | no       |
+| other_properties            | Extensions of the standard                                          | yes      |
+| version                     | Version of the NFT Identity standard                                | no       |
+
+
+
+#### 3. Identity Token
+
+This token can be created under a seperate policy ID, so that it can be used for multiple collections at once.
+The policy can also be left unlocked in case the identity information requires changes at some later date. 
+
+```
+{
+  "725": {
+    "<policy_id>": {
+        "<collection_name>": {
```

Comment:
How is the `collect_name` defined? Is there any restrictions beyond those of the protocol? Can an identity token's metadata carry multiple collections?